cypress/integration/Transfer/display-order.feature

```diff
@@ -0,0 +1,30 @@
+@component-transfer @display-ordering
+Feature: Display order of items in lists
+
+    Scenario: All supplied options are rendered in the options-side
+        Given none of the supplied options have been selected
+        Then the order of the available options should match the order of the supplied options
+
+    Scenario: Some of the supplied options have been selected
+        Given some of the supplied options have been selected
+        Then the order of the remaining available options should match the order of the supplied options
+
+    Scenario: A single selected options is deselected
+        Given some of the supplied options have been selected
+        When the user deselects one selected option
+        Then it should take its position according to the order of the supplied options
```

Comment:
`it should take its position according to the order of the supplied options`, maybe you could word this as `it should be positioned according to the order of the supplied options`. Not sure about this one, but the latter reads a bit easier for me (same as with the other comments, wording occurs in other areas as well)

Reply:
I don't really care about this one. @cooper-joe ?

Reply:
I agree with @ismay `it should be positioned according to the order of the supplied options` is better.